src/transport/raw/BLE.cpp

```diff
@@ -163,6 +163,8 @@ void BLEBase::OnEndPointConnectComplete(BLEEndPoint * endPoint, CHIP_ERROR err)
     if (err != CHIP_NO_ERROR)
     {
         ChipLogError(Inet, "Failed to establish BLE connection: %s", ErrorStr(err));
+        mState       = State::kInitialized;
```

Comment:
Minor- It might be more prudent instead to simply call OnEndPointConnectionClosed() instead of lines 166-168. Should reduce duplicity.
```suggestion
        OnEndPointConnectionClosed();
```